content/guides/models/registry/create_registry.md

```diff
@@ -7,23 +7,71 @@ title: Create a custom registry
 3. Provide a name for your registry in the **Name** field.
 4. Optionally provide a description about the registry.
 5. Select who can view the registry from the **Registry visibility** dropdown. See [Registry visibility types]({{< relref "./configure_registry.md#registry-visibility-types" >}}) for more information on registry visibility options.
 6. Select either **All types** or **Specify types** from the **Accepted artifacts type** dropdown.
 7. (If you select **Specify types**) Add one or more artifact types that your registry accepts.
+8. Click on the **Create registry** button. 
+
 {{% alert %}}
-An artifact type can not be removed from a registry once it is added and saved in the registry's settings.
+An artifact type cannot be removed from a registry once it is saved in the registry's settings.
 {{% /alert %}}
-8. Click on the **Create registry** button. 
 
 {{< img src="/images/registry/create_registry.gif" alt="" >}}
 
-For example, the preceding image shows a custom registry called "Fine_Tuned_Models" that a user is about to create. The registry is set to **Restricted** which means that only members that are manually added to the "Fine_Tuned_Models" registry will have access to this registry.
\ No newline at end of file
+For example, the preceding image shows a custom registry called `Fine_Tuned_Models` that a user is about to create. The registry is **Restricted** to only members that are manually added to the registry.
+
+
+## Visibility types
+
+The *visibility* of a registry determines who can access that registry. Restricting the visibility of a custom registry helps ensure that only specified members can access that registry.
+
+There are two type registry visibility options for a custom registry: 
+
+| Visibility | Description |
+| --- | --- | 
+| Restricted   | Only invited organization members can access the registry.| 
+| Organization | Everyone in the org can access the registry. |
+
+A team administrator or registry administrator can set the visibility of a custom registry.
+
+The user who creates a custom registry with Restricted visibility is added to the registry automatically as its registry admin. 
+
+
+## Configure the visibility of a custom registry
+
+A team administrator or registry administrator can assign the visibility of a custom registry during or after the creation of a custom registry. 
+
+To restrict the visibility of an existing custom registry:
+
+1. Navigate to the **Registry** App at https://wandb.ai/registry/.
+2. Select a registry.
+3. Click on the gear icon on the upper right hand corner.
+4. From the **Registry visibility** dropdown, select the desired registry visibility.
+
+Continue if you select **Restricted** visibility:
+
+<!-- To do: Add updated process of adding a team -->
+
+5. Add members of your organization that you want to have access to this registry. Scroll to the **Registry members and roles** section and click on the **Add member** button. 
```

Comment:
```suggestion
  1. Add members of your organization access to this registry in the **Registry members and roles** section. For each member, click **Add member**. 
```
Substeps more accurately reflect the task, and this doesn't artificially break the HTML list up into two in the output.